dask/highlevelgraph.py

```diff
@@ -404,7 +405,7 @@ def __dask_distributed_pack__(
             dependencies = {k: v - alias_keys for k, v in dependencies.items()}
         # - Add in deps for any missing keys
         missing_keys = dsk.keys() - dependencies.keys()
-        dependencies.update(
+        dependencies.update(  # type: ignore
```

Comment:
This can be fixed by adding some overloads to `keys_in_task` to get the return types right. It's okay to ignore for now, but perhaps add a TODO?